utils.py

```diff
@@ -0,0 +1,34 @@
+# autopep8: off
+import os
+import jwt
+import json
+import my_settings
+
+from pathlib                import Path
+from django.http            import JsonResponse
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.exceptions import ImproperlyConfigured
+from user.models            import User
+
+
+def signin_decorator(func):
+    def wrapper(self, request, *args, **kwargs):
+        try:
+            token     = request.headers.get('Authorization', None)
+            key       = my_settings.SECRET.get('JWT_KEY')
+            algorithm = my_settings.SECRET.get('JWT_ALGORITHM')
+
+            if token:
+                decode       = jwt.decode(token, key, algorithm=algorithm)
+                user         = User.objects.get(id=decode['user'])
+                request.user = user
+            else :
+                return JsonResponse({"message" : "NO_TOKEN"}, status=403)
```

Comment:
```suggestion
                return JsonResponse({"message" : "TOKEN_DOES_NOT_EXIST"}, status=403)
```
문법상으로는 이부분이 상단조건으로 가고 아래에 정상처리 로직을 하면 else를 쓰지 않아도 될것 같습니다.

Reply:
넵 변경했습니다:)